### PR TITLE
Database queries made. Routes dynamic configurable

### PR DESCRIPTION
**Pull request by kyunwang**
- End time updated query on database
- Dynamic return routes for back button made configurable
- Complete and cancel calls for tour to db
- Minor change exampletour

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const exampleTourComplete = [
 		origins: 'Vincent van Gogh (1853 - 1890), Antwerpen, januari-"februari" 1886',
 		description:
 			'Dit skelet met een brandende sigaret is een studentikoze grap. Van Gogh schilderde het in de tijd dat hij lessen volgde aan de Antwerpse kunstacademie, begin 1886. Het schilderij laat zien dat hij de anatomie goed beheerste.',
-		visited: false,
+		// visited: false,
```
Comment (VincentKempers) [inline src/api/index.js]: Do we still need this then!?

File: src/views/Apptour/Tourmap.vue [hunk 2]
```diff
@@ -224,24 +240,28 @@ main {
 		position: fixed;
 		right: 0;
 		margin: 0 1em 0em 0;
-		bottom: -4em;
+		// bottom: -4em;
```
Comment (VincentKempers) [inline src/views/Apptour/Tourmap.vue]: delete this

File: src/views/Apptour/Tourmap.vue [hunk 3]
```diff
@@ -224,24 +240,28 @@ main {
...
+		// bottom: 1em;
+		bottom: 1em;
```
Comment (VincentKempers) [inline src/views/Apptour/Tourmap.vue]: EVEN YOU DON'T USE REM ANYMORE 🤔

Comment (kyunwang) [reply src/views/Apptour/Tourmap.vue]: Readability. No all caps because no one will want to read it. And I have changed it now👍

**Pull request merged**